_site/docs/architecture/content_addressable_storage.md

```diff
@@ -34,7 +34,7 @@ Since these implementations vary in complexity and storage semantics, a common i
 
 The memory CAS implementation is extremely simple in design, constituting a maximum size with LRU eviction policy. Entry eviction is a registrable event for use as a storage for the delegated ActionCache, and Writes may be completed asynchronously by concurrent independent upload completion of an entry.
 
-This is the example presentation of a CAS in the memory instance available [here](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/server.config.example#L40-L45), but for reference, specification in any `cas_config` field for server or worker will enable the creation of a unique instance.
+This is the example presentation of a CAS in the memory instance available [here](https://github.com/bazelbuild/bazel-buildfarm/blob/85c3fdaf89fedc2faee1172fab01338777de79a1/examples/config.memory.yml#L40-L45), but for reference, specification in any `cas_config` field for server or worker will enable the creation of a unique instance.
```

Comment:
probably address these doc changes in a followup PR because `config.memory.ym` file won't exist in the snapshot on a previous commit.

Reply:
that's a good point.  Our generated documentation has no version picker.  Whatever lands on main will become the live site.